privacy/scrubber.go

```diff
@@ -260,6 +262,12 @@ func scrubIPV6Lowest16Bits(ip string) string {
 	return ip
 }
 
+func anonymizeIpv6(ip string) string {
+	ipv6Mask := net.CIDRMask(56, 128)
```

Comment:
To match PBS-Java, these two variables should be read from the config (account config?) with appropriate validation on the values.

Reply:
In progress, not yet ready for review

Reply:
Moved to `account.privacy`